pkg/apis/hive/v1alpha1/syncsetinstance_types.go

```diff
@@ -82,6 +82,9 @@ type SyncSetInstanceStatus struct {
 	// when a resource type cannot be determined from a SyncSet resource.
 	// +optional
 	Conditions []SyncCondition `json:"conditions,omitempty"`
+
+	// Applied will be true if all resources, patches, or secrets have successfully been applied on last reconcile.
```

Comment:
This should be "on the last attempt to apply them" rather than "on last reconcile". There could be reconciles without applies.

Reply:
Updated with a slightly more brief version.